assets/device/classes/speaker.json

```
@@ -18,7 +20,9 @@
     "sv": "Använd den här enhetsklassen för enheter som kan spela musik, oftast tilsammans med funktionerna `högtalare_*`.",
     "no": "Bruk denne enhetsklassen for enheter som kan spille musikk, ofte sammen med funksjonen `høyttaler_*`.",
     "es": "Utiliza esta clase de dispositivo para los aparatos capaces de reproducir música, generalmente junto con las capacidades 'altavoz_*'.",
-    "da": "Brug denne enhedsklasse til enheder, der kan afspille musik, normalt sammen med `højttaler_*` funktionerne."
+    "da": "Brug denne enhedsklasse til enheder, der kan afspille musik, normalt sammen med `højttaler_*` funktionerne.",
```

```suggestion
    "da": "Brug denne enhedsklasse til enheder, der kan afspille musik, normalt sammen med `speaker_*` funktionerne.",
```

This should also be fixed for other translations in this description